include/mbedtls/ssl.h

```diff
@@ -2043,6 +2043,14 @@ void mbedtls_ssl_conf_ca_chain( mbedtls_ssl_config *conf,
  *                 whether it matches those preferences - the server can then
  *                 decide what it wants to do with it.
  *
+ * \note           The provided \p pk_key needs to match the public key in the
+ *                 first certificate in \p own_cert, or all handshakes using
+ *                 that certificate will fail. It is your reponsibility
+ *                 to ensure that; this function will not perform any check.
```

Comment:
I believe the semicolon(`;`) is a typo

Reply:
I think the semicolon is justified here, as the line of thought continues.

Reply:
yes, the line of thought continues. I think there is no need for the semicolon at all

Reply:
There's not much ground for an argument here, it's a matter of taste. Let's have @mpg decide it.

Reply:
I agree

Reply:
FWIW, the semicolon was intentional.